db/migrate/20140929233719_create_events.rb

```diff
@@ -1,4 +1,8 @@
 class CreateEvents < ActiveRecord::Migration
+  class Event < ActiveRecord::Base
+    translates :title, :introduction, :conclusion
+  end
```

Comment:
The globalize translations are introduced in this migration then later removed, this causes an error in the  `rake db:migrate` .  This is a workaround to the issue @nicholasjhenry reported.